mp/src/game/shared/momentum/mom_system_gamemode.h

```diff
@@ -72,6 +80,7 @@ class CGameMode_RJ : public CGameModeBase
     const char* GetStatusString() override { return "Rocket Jumping"; }
     const char* GetDiscordIcon() override { return "mom_icon_rj"; }
     const char* GetMapPrefix() override { return "jump_"; }
+    const char* GetGameModeCfg() override { return "exec rj.cfg"; }
```

Comment:
Something to consider for the future, RJ will technically need two configs, one for soldier and one for demo.

Reply:
I don't know how to do that currently, with demoman not being in the game

Reply:
It will also be the only game mode where this happens, so yeah it's out of scope of this pr